main.tf

```diff
@@ -0,0 +1,91 @@
+/**
+* Module usage:
+*
+*      module "msk_cluster" {
+*        source = "git::https://github.com/UKHomeOffice/acp-tf-msk-cluster?ref=master"
+*
+*        name                   = "msktestclutser"
+*        msk_instance_type      = "kafka.m5.large"
+*        kafka_version          = "1.1.1"
+*        environment            = "${var.environment}"
+*        number_of_broker_nodes = "3"
+*        subnet_ids             = ["${data.aws_subnet_ids.suben_id_name.ids}"]
+*       vpc_id                 = "${var.vpc_id}"
+*        ebs_volume_size        = "50"
+*        cidr_blocks            = ["${values(var.compute_cidrs)}"]
+*      }
+*
+*
+*
+ */
+data "aws_caller_identity" "current" {}
+
+data "aws_region" "current" {}
+
+resource "aws_security_group" "sg_msk" {
+  name        = "kafka_security_group"
```

Comment:
This should have the cluster name in it too (otherwise you won't be able to have multiple in the same account)